canonical-web-and-design/maas-ui
# Commit vscode workspace configuration.

## Done
Commit vscode workspace configuration, with configuration for enabling eslint in the monorepo.

## QA
* Restart your VSCode with the new file in place.
* Open up a `js` file in the project, and break something. You should see squiggles and eslint warnings inline.
